frontend/src/metabase/admin/tools/components/Logs/utils.ts

```diff
@@ -99,11 +98,15 @@ export function getAllProcessUUIDs(logs: Log[]) {
 const formatTs = (ts: string) => dayjs(ts).format();
 const memoedFormatTs = _.memoize(formatTs);
 
-export function formatLog(log: Log) {
-  const timestamp = memoedFormatTs(log.timestamp);
-  const uuid = log.process_uuid || "---";
-  return [
-    `[${uuid}] ${timestamp} ${log.level} ${log.fqns} ${log.msg}`,
-    ...(log.exception || []),
-  ];
-}
+export const formatLog =
```

Comment:
this function name no longer really makes sense, it should probably be called `createLogFormatter`